plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/MultiParentCompileHook.java

```diff
@@ -117,7 +117,11 @@ private MavenRunner.Config getMavenConfig(PluginCompatTesterConfig config) throw
     }
 
     private void compile(MavenRunner.Config mavenConfig, File path, File localCheckoutDir, String parentFolder, String pluginName) throws PomExecutionException, IOException {
-        if (isSnapshotMultiParentPlugin(parentFolder, path, localCheckoutDir)) {
+        if (pluginName.equals("workflow-cps")) {
+            // TODO: Why does the following condition not work for workflow-cps? Is it only broken for local checkouts?
```

Comment:
> Maybe it is just the case that `-localCheckoutDir` has never worked correctly with multi-module projects?

I would not be surprised. Everything about `MultiParentCompileHook` is dumb. This part of PCT should be rewritten so that no hooks are needed—should collect repos corresponding to selected plugins, then run `mvn` commands from repo root.

Reply:
https://github.com/jenkinsci/plugin-compat-tester/issues/384 tracks the `-localCheckoutDir` issue.